.github/labeler.yaml

```diff
@@ -0,0 +1,36 @@
+area/docs:
+- "docs/**/*"
+- "**/*.md"
+
+area/config:
+- "config/*"
+- "config/**/*"
+
+area/installer:
+- "agent-installer/**/*"
+
+area/apis:
+- "apis/*"
+- "apis/**/*"
+
+area/test-release:
+- "hack/**/*"
+- "test/**/*"
+- "go.mod"
+- "Makefile"
+
+area:
+- "apis"
+- "docs"
+- "test-release"
+- "installer"
+- "config"
+
+kind:
+- "bug"
+- "feature"
+- "enhancement"
+- "docs"
+- "test-release"
+- "feedback"
+- "config"
```

Comment:
Suggestion: I guess we do not require 1-1 mapping of the label area to label kind. It could be another list with values something like below.
- bug
- feature
- cleanup

Reply:
Make sense. What else we can add?